hazelcast/src/main/resources/hazelcast-config-5.1.json

```diff
@@ -291,6 +291,11 @@
           "type": "boolean",
           "default": false
         },
+        "data-access-enabled": {
+          "description": "Set to true to allow data access for Management Center execution on the member, false to disallow. Management Center can't access the data if at least one member has the data access disabled.",
```

Comment:
> allow data access for Management Center execution on the member

"execution" is unnecessary. Also it would make sense to elaborate on what "data" means in this context.

Suggestion: "Set it to false to prohibit Management Center from accessing contents of Hazelcast data structures (eg. map entries)."

Reply:
agree, updated